### DISCUSSION TITLE
Esp32-C2 support

### DISCUSSION THREAD

**Discussion by elimisback** [arendst/Tasmota#17330] [Ideas]
Just throwing it out there to see if there was any plan to support esp32-c2 devices.  I tried seeing if the c3 would just work but no luck. Hope everyone is having a good holiday season!

**Comment by s-hadinger** [comment]
Yes there are plans. But we need also Arduino support. I'm not sure yet which version is required for this chip

**Comment by Jason2866** [comment]
Will not be supported (soon), since as already written is needs to be supported from Arduino.
Since the C2 is first supported in IDF 5.0.x a new Arduino version is needed which is based on IDF 5.0.
The actual Arduino version (2.0.5) is based on IDF 4.4. The next Arduino release will be 2.0.6 (still based on IDF 4.4). After this release Arduino will be based on IDF 5.0. with version 3.0.0. There is no time target from espressif for release of Arduino v3.0.0

**Comment by elimisback** [comment]
Thanks for the info!

**Comment by platima** [comment]
Hey folks just an update - Espressif have said that the C2 won't likely be supported by Arduino aka no internal development effort. Ref: https://github.com/espressif/arduino-esp32/issues/6701#issuecomment-1156377493

Sad because I'd be happy with ESP Home, Arduino or Tasmota support :(
